MvvmCross/Binding/BindingContext/MvxBaseFluentBindingDescription.cs

```diff
@@ -163,6 +163,12 @@ public static class SourceSpecHelpers
 
         protected object ClearBindingKey { get; set; }
 
+        object IMvxBaseFluentBindingDescription.ClearBindingKey
```

Comment:
This should be public

Reply:
@nmilcoff  I believe the default access modifier of an explicit interface implementation is public by virtue of the interface contract require it to be public. As it can only be public C# does not require an access modifier to be specified ([Explicit Interface Implementation](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/interfaces/explicit-interface-implementation)).

Reply:
Oh, right! I didn't see it was implemented explicitly 🙂 👍

Reply:
No worries 🙂